pkg/tfbridge/provider.go

```diff
@@ -1237,7 +1237,7 @@ func (p *ProviderInfo) RenameResourceWithAlias(resourceName string, legacyTok to
 			currentInfo.Tok.Name().String()))
 	p.Resources[resourceName] = &currentInfo
 	p.Resources[legacyResourceName] = &legacyInfo
-	p.P.ResourcesMap().Set(legacyResourceName, p.P.ResourcesMap().Get(resourceName))
+	p.P.ResourcesMap().AddAlias(legacyResourceName, resourceName)
```

Comment:
This and the change below seem to eliminate the runtime use of Set on the bridge

Reply:
Can we save this observation in a ticket? That'd be a good change to deprecate Set - except for this use case, which can be better named.